cargo-dist/src/backend/ci/github.rs

```diff
@@ -116,6 +119,21 @@ impl GithubCiInfo {
             .hosts
             .clone();
 
+        let release_phase = if dist.github_release == GithubReleasePhase::Auto {
+            // If Axo is in use, we want the release to come late
+            // because the release wil lcontain links to Axo URLs
+            // that won't become live until the announce phase.
+            if hosting_providers.contains(&HostingStyle::Axodotdev) {
+                GithubReleasePhase::Announce
+            // Otherwise, it's safe for the release to occur during host.
```

Comment:
That said imo the primary framing is that we *want* to do the Github release in announce, and that axo releases makes it safe to do so. So this comment feels backwards. Putting it in Host is the "do it for safety" option.

Reply:
Fair enough, yeah - I can reverse the comments. I've been thinking about the npm releases, which made me think of "do it in host" as the ideal and announce as the safety when Axo is used.

Reply:
Updated comments.